src/timeunit.ts

```diff
@@ -240,6 +241,12 @@ export type TimeUnitFormat =
   | 'seconds'
   | 'milliseconds';
 
+export interface TimeUnitParams {
+  units: TimeUnit;
```

Comment:
Missing docs. I'd move the docs you define for TimeUnitTranform here and use this TimeUnitParams as mixins for `TimeUnitTranform`'s declaration.

Reply:
done